templates/commands/render/render_action_include.go

```diff
@@ -68,6 +66,23 @@ func includePath(ctx context.Context, inc *spec.IncludePath, sp *stepParams) err
 			return err
 		}
 
+		// By default, we copy from the template directory. We also support
+		// grabbing files from the destination directory, so we can modify files
+		// that already exist in the destination.
+		fromDir := sp.templateDir
+		if inc.From.Val == "destination" {
+			fromDir = sp.flags.Dest
+		}
+
+		absSrcPaths, err := filepath.Glob(filepath.Join(fromDir, walkRelPath))
```

Comment:
For a followup PR, it would be cool to add a debug log saying eg `glob *.txt expanded to [a.txt, b.txt]`

Reply:
Added debug logging to processGlobs.